pages/open_cas_linux/guide_monitoring_json.md

```diff
@@ -0,0 +1,497 @@
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Read partial misses": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Read full misses": {
+      "Page": 43,
+      "Fraction": 10000
+    },
+    "Read total": {
+      "Page": 43,
+      "Fraction": 10000
+    },
+    "Write hits": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Write partial misses": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Write full misses": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Write total": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Pass-Trough reads": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Pass-Trough writes": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Serviced requests": {
+      "Page": 43,
+      "Fraction": 10000
+    },
+    "Total requests": {
+      "Page": 43,
+      "Fraction": 10000
+    }
+  },
+  "Blocks": {
+    "Reads from core(s)": {
+      "Page": 259,
+      "Fraction": 10000
+    },
+    "Writes from core(s)": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Total from/to core(s)": {
+      "Page": 259,
+      "Fraction": 10000
+    },
+    "Reads from cache(s)": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Writes from cache(s)": {
+      "Page": 259,
+      "Fraction": 10000
+    },
+    "Total from/to cache(s)": {
+      "Page": 259,
+      "Fraction": 10000
+    },
+    "Reads from exported object(s)": {
+      "Page": 259,
+      "Fraction": 10000
+    },
+    "Writes from exported object(s)": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Total from/to exported object(s)": {
+      "Page": 259,
+      "Fraction": 10000
+    }
+  },
+  "Errors": {
+    "Core read errors": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Core write errors": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Core total errors": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Cache read errors": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Cache write errors": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Cache total errors": {
+      "Page": 0,
+      "Fraction": 0
+    },
+    "Total errors": {
+      "Page": 0,
+      "Fraction": 0
+    }
+  }
+}
+```
+
+# List caches
+
+Checking CAS devices with casadm list caches mentioned in [**Monitoring CAS**](/guide_monitoring.html).
+
+```console
+[root@localhost json]# casadm -L
+\type    id   disk             status    write policy   device
+cache   1    /dev/nvme3n1p4   Running   wt             -
+├core   1    /dev/nvme0n1p3   Active    -              /dev/cas1-1
+└core   2    /dev/nvme0n1p4   Active    -              /dev/cas1-2
+```
+
+# Example list caches request
+
+```json
+{
+  "header": {
+      "command": "cache.get_list"
+  },
+  "params": {
+  }
+}
+```
+
+# Example list caches response
+
+**Example List caches response structure:**
+
+-   for each cache instance:
+    -   Cache details
+    -   for each core in cache:
+        -   Core details
+```json
+[
```

Comment:
do we want to have all the information here? cache.get_list has lots of information from cache.info.get

Reply:
Most likely limiting info in list caches to minimum is good way to go.